fix VCF export END and validate lines (#5813)
Clinical-Genomics/scout: CHANGELOG.md

- Deprecate python 3.9 which has reached its EOL, default build to 3.14 (#5818)
- Don't build in editable mode for Docker images (#5819)
- Restored installation of chanjo-report from PyPI (#5820)
- Export of managed variants to VCF is subject to validation. variants failing validation will raise a warning instead of being included in the VCF file (#5813)
### Fixed
- Filter `f` hotkey not working on cancer variantS pages (#5788)
- IGV.js updated to v3.5.4 (#5790)
- Broken ClinVar germline submission page when submission contains SVs (#5805)
- Missing size_selector in old filters containing a size criterion, causing the variantS page to crash (#5817)
- Avoid backslash in f-strings, to preserve backwards compatibility with old python (#5818)

> - GnomAD links for the STR variant page (#5823)- Empty string as END position is not longer accepted when exporting manages variants to VCF file. END will be omitted for SNVs instead (#5813)

Reviewer: ```suggestion
- GnomAD links for the STR variant page (#5823)
- Empty string as END position is no longer accepted when exporting managed variants SVs to VCF file. INFO.END will be omitted for SNVs (#5813)
```

- Avoid backslash in f-strings, to preserve backwards compatibility with old python (#5818)
- GnomAD links for the STR variant page (#5823)
- `get_gene_complete_coverage` breaking for genes with no intervals on chanjo2, which is returning 'NA' instead of numerical stats (#5830)

> - Empty string as END position is no longer accepted when exporting managed variants SVs to VCF file. INFO.END will be omitted for SNVs (#5813)

Author: I'll rephrase!
